[new product] opa gatekeeper (#5014)
captn3m0/endoflife.date: products/opa-gatekeeper.md

---
title: opa-gatekeeper

> category: app

Reviewer: ```suggestion
category: server-app
```

tags: cncf configuration-management
permalink: /opa-gatekeeper

> alternate_urls:
> -   /opa-gatekeeper

Reviewer: ```suggestion
```

releasePolicyLink: https://github.com/open-policy-agent/gatekeeper/blob/master/docs/Release_Management.md#supported-releases
changelogTemplate: https://github.com/open-policy-agent/gatekeeper/releases/tag/v__LATEST__
releaseDateColumn: true
eolColumn: Support

auto:
  methods:
  -   git: https://github.com/open-policy-agent/gatekeeper.git

>       regex: ^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(-ce)?$

Reviewer: I didn't see any tags ending with `-ce` so the default regex should just work for this?

```suggestion
```

> 

Reviewer: ```suggestion

- identifiers:
-   repology: gatekeeper
-   purl: pkg:github/open-policy-agent/gatekeeper
-   purl: pkg/docker/openpolicyagent/gatekeeper
-   purl: pkg/docker/chainguard/gatekeeper
-   purl:oci/gatekeeper?repository_url=cgr.dev/chainguard

```

releases:
-   releaseCycle: "3.15"
    releaseDate: 2024-02-05
    eol: false
    latest: "3.15.1"
    latestReleaseDate: 2024-03-12

-   releaseCycle: "3.14"
    releaseDate: 2023-11-01
    eol: false
    latest: "3.14.1"
    latestReleaseDate: 2024-03-12

-   releaseCycle: "3.13"
    releaseDate: 2023-08-09
    eol: 2023-11-01
    latest: "3.13.4"
    latestReleaseDate: 2023-11-02

-   releaseCycle: "3.12"
    releaseDate: 2023-04-14
    eol: 2023-08-09
    latest: "3.12.0"
    latestReleaseDate: 2023-04-14

-   releaseCycle: "3.11"
    releaseDate: 2023-01-04
    eol: 2023-04-14
    latest: "3.11.1"
    latestReleaseDate: 2023-04-26

---

> > [opa-gatekeeper](https://open-policy-agent.github.io/) Gatekeeper - Policy Controller for Kubernetes

Reviewer: ```suggestion
> [OPA Gatekeeper](https://open-policy-agent.github.io/gatekeeper/website/) - is a policy controller for Kubernetes that helps enforce policies and strengthen governance.
```

> Supported versions are (current) and n-1 major.minor releases

Reviewer: ```suggestion
The latest two releases are supported with bug and security fixes.
```

---

> title: opa-gatekeeper

Reviewer 2: ```suggestion
title: Gatekeeper
```

This product is usually named Gatekeeper on https://github.com/open-policy-agent/gatekeeper/tree/master?tab=readme-ov-file#how-is-gatekeeper-different-from-opa and https://open-policy-agent.github.io/gatekeeper/website/docs/howto/.

category: server-app

> tags: cncf configuration-management

Reviewer 2: ```suggestion
tags: cncf
```

configuration-management is [not an existing tag](https://endoflife.date/tags/).

> permalink: /opa-gatekeeper

Reviewer 2: ```suggestion
permalink: /gatekeeper
```

This product is usually named Gatekeeper on https://github.com/open-policy-agent/gatekeeper/tree/master?tab=readme-ov-file#how-is-gatekeeper-different-from-opa and https://open-policy-agent.github.io/gatekeeper/website/docs/howto/.

releasePolicyLink: https://github.com/open-policy-agent/gatekeeper/blob/master/docs/Release_Management.md#supported-releases
changelogTemplate: https://github.com/open-policy-agent/gatekeeper/releases/tag/v__LATEST__

> releaseDateColumn: true

Reviewer 2: ```suggestion
```

This is now the default.

eolColumn: Support

auto:
  methods:
  -   git: https://github.com/open-policy-agent/gatekeeper.git

identifiers:
-   repology: gatekeeper
-   purl: pkg:github/open-policy-agent/gatekeeper
-   purl: pkg/docker/openpolicyagent/gatekeeper
-   purl: pkg/docker/chainguard/gatekeeper

> -   purl:oci/gatekeeper?repository_url=cgr.dev/chainguard

Reviewer 2: ```suggestion
-   purl: oci/gatekeeper?repository_url=cgr.dev/chainguard
```

:lipstick:

releases:
-   releaseCycle: "3.15"
    releaseDate: 2024-02-05
    eol: false
    latest: "3.15.1"
    latestReleaseDate: 2024-03-12

-   releaseCycle: "3.14"
    releaseDate: 2023-11-01
    eol: false
    latest: "3.14.1"
    latestReleaseDate: 2024-03-12

-   releaseCycle: "3.13"
    releaseDate: 2023-08-09
    eol: 2023-11-01
    latest: "3.13.4"
    latestReleaseDate: 2023-11-02

-   releaseCycle: "3.12"
    releaseDate: 2023-04-14
    eol: 2023-08-09
    latest: "3.12.0"
    latestReleaseDate: 2023-04-14

-   releaseCycle: "3.11"
    releaseDate: 2023-01-04
    eol: 2023-04-14
    latest: "3.11.1"
    latestReleaseDate: 2023-04-26

---

> > [OPA Gatekeeper](https://open-policy-agent.github.io/gatekeeper/website/) is a policy controller for Kubernetes that
> > helps enforce policies and strengthen governance.

Reviewer 2: ```suggestion
> [Gatekeeper](https://open-policy-agent.github.io/gatekeeper/website/) is a policy controller for Kubernetes that
> helps enforce policies and strengthen governance.
```
